composer.json

```diff
@@ -18,7 +18,7 @@
     "ext-json": "*",
     "ext-openssl": "*",
     "google/apiclient": "^2.0",
-    "google/apiclient-services": "~0.1",
+    "google/apiclient-services": ">=0.156",
```

Comment:
@benbowler @aaemnnosttv Shouldn't this be `>0.156` instead? Not a big deal for now though - alternatively we can wait for the new version to have the actual version number and then use `>=` with that one instead.